hypothesis-python/tests/common/debug.py

```diff
@@ -18,7 +18,7 @@
 
 from tests.common.utils import no_shrink
 
-TIME_INCREMENT = 0.001
+TIME_INCREMENT = 0.0001
```

Comment:
Say more about what prompted this?

Reply:
`NodeTemplate("simplest", ...)` uses `buffer_to_ir` which instantiates `ConjectureData` which calls `time.time()`, invoking `_consistently_increment_time` even though little work is actually happening. post-IR we should be able to revert this.